modules/client-configuration/pages/creating-a-tools-repository.adoc

```diff
@@ -24,6 +24,9 @@ You will need to check all the base and child channel log files to be sure that
 
 == Generate a Tools Repository
```

Comment:
I think the name of the file and the title is wrong.

The "Tool repository" is SUSE Manager Tools or Client Tools. This is also a product. It is different to the bootstrap repository. It is dangerous to mix them.

The Bootstrap Repository contains packages from the Tools Repository, but not all of them. And it contains also packages from other repos.

Please change all the names and titles here and name it "Bootstrap Repository".

Reply:
I'm really glad you said this, I've been less than happy about the title of this page for some time. Now let's see which xrefs break!

Reply:
yes, thanks!  I unfortunately often confuse them  :-(